README.md

```diff
@@ -1001,3 +1001,34 @@ distributions:
   - `sign-key-name: str` --- name of the signing key to use. For test keys this becomes the subject of the self-signed certificate.
   - `test-sign: bool` --- code signing type, `true` (default) or `false`. Test signing generates ephemeral self-signed
     keys for each component. Production signing uses an already existing key signed by a public CA (TODO: HSM).
+
+### Cross-distribution dependencies
+
+To perform cross-compilation between distributions, you must declare dependencies using the `needs: List[Dict]` structure within the appropriate distribution stage.
+Each dependency is represented as a dictionary with the following keys:
+
+- `component: str` ---  The name of the component. This value is not limited to the top-level component reference; it can reference any available component.
+- `distribution: str` --- The name of the target distribution.
+- `stage: str` --- The stage name for which the dependency is required.
+- `build: str` --- The build reference as provided in a `.qubesbuilder` file.
+
+For example:
+
+```yaml
+components:
+  - installer-qubes-os-windows-tools:
+      host-fc41:
+        stages:
+          - build:
```

Comment:
isn't it prep?

Reply:
this one is still relevant

Reply:
If this is prep then I can change, I just did not remembered and thought that would changed when actually being done.

Reply:
I think source commands are running in prep stage, like you did in the test component.

Reply:
I updated the example to have `prep` phase for `host-fc41` these needs.